packages/flame/example/lib/main.dart

```diff
@@ -14,14 +14,51 @@ void main() {
   );
 }
 
-class Square extends PositionComponent {
+/// This example simply adds a rotating white square on the screen, if you press
+/// somewhere other than on the existing square another square will be added and
+/// if you press on a square it will be removed.
+class MyGame extends FlameGame with DoubleTapDetector, HasTappables {
+  bool running = true;
+
+  @override
+  Future<void> onLoad() async {
+    await super.onLoad();
+    add(Square(Vector2(100, 200)));
+  }
+
+  @override
+  void onTapUp(int id, TapUpInfo info) {
+    super.onTapUp(id, info);
+    final touchPoint = info.eventPosition.game;
+    final handled = children.any((c) => c.shouldRemove);
+
+    if (!handled) {
+      add(Square(touchPoint));
+    }
+  }
+
+  @override
+  void onDoubleTap() {
```

Comment:
this seems like a very specific feature to showcase on this example

Reply:
It was here already, I just moved the class.